### PR TITLE
Update PHPCS config, run PHPCS checks as a part of the CI build

### PR DESCRIPTION
**Pull request by manicki**

### CODE REVIEW

**Comment by Ladsgroup** [comment]
The failure was not related and perhaps was because of github outage today. I will fix it if it's really broken.

File: DataTypes.php [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
 namespace DataTypes;
 
-if ( defined( 'DataTypes_VERSION' ) ) {
+if ( defined( 'DATATYPES_VERSION' ) ) {
 	// Do not initialize more than once.
 	return 1;
 }
 
-define( 'DataTypes_VERSION', '1.0.0' );
+define( 'DATATYPES_VERSION', '1.0.0' );
```
Comment (thiemowmde) [inline DataTypes.php]: This is a breaking change. I don't think this alone justifies a 2.0.0 release. See my suggestion in #70 to solve this issue.

Comment (manicki) [inline DataTypes.php]: meh, I'd ignore the sniff then. But whatever.

Comment (thiemowmde) [inline DataTypes.php]: Huh? Which sniff?

Comment (manicki) [inline DataTypes.php]: Generic.NamingConventions.UpperCaseConstantName.ConstantNotUpperCase

Comment (manicki) [inline DataTypes.php]: it was the reason why it was changed uppercase

Comment (thiemowmde) [inline DataTypes.php]: Uh, what's going on here? This check fails now, but Travis doesn't report it: https://travis-ci.org/wmde/DataTypes/jobs/259698637

**Pull request merged**